tests/compile_commands.json

```diff
@@ -0,0 +1,212 @@
+[
```

Comment:
I think this file probably should not be part of the PR, right? Can you remove it?